appengine-java8/spanner/pom.xml

```diff
@@ -45,7 +45,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>libraries-bom</artifactId>
-        <version>5.4.0</version>
```

Comment:
Why was this 5.4.0, not 5.7.0?

Reply:
That was the last version that passed.  We are trying to get the spanner team to fix.